src/files.h

```diff
@@ -4,4 +4,12 @@
 #include <string>
 #include <vector>
 
-std::vector<char> gulp(std::string fname);
+std::vector<char> gulp(std::string fname) {
```

Comment:
Seeing as you are touching this code anyways, I would like to see a function name that is a bit more descriptive than `gulp`.

Reply:
I've named it to `readFile` and added some documentation, let me know if you think this is OK.